ftest/web-ui/webdriver/itests.xml

```diff
@@ -16,6 +16,7 @@
       <artifact:file key="org.nuxeo.web.ui:nuxeo-web-ui-marketplace::zip" />
       <artifact:file key="org.nuxeo.web.ui:nuxeo-web-ui-marketplace-itests::zip" />
     </copy>
+    <copy todir="${nuxeo.home}/nxserver/data" file="instance.clid" />
```

Comment:
Do we already have an instance.clid file? 🤔

Reply:
Ah, I see this is related to https://github.com/nuxeo/nuxeo-web-ui/pull/1379. What happens if we run this locally and we don't have a instance.clid file?

Reply:
I added the instance clid in the pipeline https://github.com/nuxeo/nuxeo-web-ui/pull/1379/files 🤔  We shouldn't have the clid in the repository directly

Reply:
If there's no clid, then it runs the tests without registering the server.